Formula/d/devcockpit.rb

```diff
@@ -0,0 +1,30 @@
+class Devcockpit < Formula
+  desc "TUI system monitor for Apple Silicon"
+  homepage "https://devcockpit.app/"
+  url "https://github.com/caioricciuti/dev-cockpit/archive/refs/tags/v1.0.8.tar.gz"
+  sha256 "a1ce6d16d46da379d88ca579f24d9d16c542b047c6dd3005637c2d45cf7c49e7"
+  license "GPL-3.0-only"
+  head "https://github.com/caioricciuti/dev-cockpit.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on arch: :arm64
```

Comment:
Does it work on arm64 Linux?

Reply:
From the [docs](https://devcockpit.app/getting-started.html#prerequisites):
> Prerequisites
> macOS 11.0 (Big Sur) or later

Reply:
@stefanb the big_sur requirement is not needed, as the first macos release for apple silicon is big_sur

Reply:
if linux builds are useless and not supported by upstream they should be prevented here by limiting to macos.